### PR TITLE
Feat/eia 471 improve virus tests

### PR DESCRIPTION
**Pull request by RichardBray**
# Description

This PR predominantly adds tests to make sure that if virus and file type checking was removed then the tests would fail.

It also fixes an issue where file checks where happening on previously uploaded files

Screenshot 2022-05-13 at 15 20 16

### CODE REVIEW

File: tests/specs/controllers/FileUpload.test.js [hunk 1]
```diff
@@ -2,15 +2,23 @@ const request = require('supertest');
...
+
+const expect = chai.expect;
 const FileUploadController = require('../../../api/controllers/FileUploadController');
 const HelperService = require('../../../api/services/HelperService');
 const Application = require('../../../api/models/index').Application;
 
 const sandbox = sinon.sandbox.create();
 
+function assertWhenPromisesResolved(assertion) {
+    setTimeout(assertion);
```
Comment (PaulioRandall) [inline tests/specs/controllers/FileUpload.test.js:19]: Isn't setTimeout supposed to have a time in ms as the 2nd argument?

**Review by PaulioRandall — COMMENTED** [review]
Just the one minor thing

File: tests/specs/controllers/FileUpload.test.js (hunk 1 above)
Comment (RichardBray) [reply tests/specs/controllers/FileUpload.test.js:19]: Not necessarily, the second argument is set to 0 by default
https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#parameters

This is sort of a hacky way to trick the runner to run this assertion after the other functions have been executed in the call stack.

Comment (PaulioRandall) [reply tests/specs/controllers/FileUpload.test.js:19]: Ah, a deferred execution. Nice trick.

**Review by PaulioRandall — APPROVED** [review]

**Pull request merged**